lib/vagrant-libvirt/action/forward_ports.rb

```diff
@@ -93,7 +93,7 @@ def redirect_port(machine, host_ip, host_port, guest_ip, guest_port,
             -N
             #{ssh_info[:host]}
           )
-          params += '-g' if gateway_ports
+          params <<= '-g' if gateway_ports
```

Comment:
while I was familiar with with `<<` I had to check about `<<=`, thanks for the fix